src/rp2_common/pico_platform/include/pico/platform.h

```diff
@@ -153,14 +153,14 @@ extern "C" {
  *
  * For example a `uint32_t` foo that will retain its value if the program is restarted by reset.
  *
- *     uint32_t __uninitialized_ram("my_group_name") foo;
+ *     uint32_t __uninitialized_ram("foo");
  *
  * The section attribute is `.uninitialized_ram.<group>`
  *
  * \param group a string suffix to use in the section name to distinguish groups that can be linker
  *              garbage-collected independently
  */
-#define __uninitialized_ram(group) __attribute__((section(".uninitialized_ram." #group))) group
+#define __uninitialized_ram(group) __attribute__((section(".uninitialized_data." #group))) group
```

Comment:
@kilograham This is the "change a few lines below" that I was referring to in my previous comment, but like I said I don't really understand all this stuff.

Looking at the other defines in this file, is line 177 missing a `.` off the end of the name, i.e. should `".flashdata"` be `".flashdata."` ? :shrug: